ccm_web/tsconfig.json

```diff
@@ -24,6 +25,6 @@
     "server",
```

Comment:
the `include` property don't warn if files other than `js/ts` are asked to take into consideration like `sh` or hidden files. 

But I have used the `files property and error out in the logs 

```
 error TS6054: File '/build/.sequelizerc' has an unsupported extension. The only supported extensions are '.ts', '.tsx', '.d.ts', '.js', '.jsx'.
#16 53.49   The file is in the program because:
#16 53.49     Part of 'files' list in tsconfig.json
#16 53.49 error TS6054: File '/build/server/.sequelizerc' has an unsupported extension. The only supported extensions are '.ts', '.tsx', '.d.ts', '.js', '.jsx'.
#16 53.49   The file is in the program because:
#16 53.49     Part of 'files' list in tsconfig.json
#16 53.49 error TS6054: File '/build/start.sh' has an unsupported extension. The only supported extensions are '.ts', '.tsx', '.d.ts', '.js', '.jsx'.
#16 53.49   The file is in the program because:
#16 53.49     Part of 'files' list in tsconfig.json
```